### PR TITLE
APIM: Open port for core services of the management API

### PR DESCRIPTION
**Pull request by gaetanmaisse**
**Issue**

https://github.com/gravitee-io/issues/issues/6895

**Description**

Open service core port for management API as it was already done for the gateway.

### CODE REVIEW

**Comment by brasseld** [comment]
Can you please add a test for this ?

**Comment by gaetanmaisse** [comment]
@brasseld Done ✅

File: apim/3.x/templates/api/api-deployment.yaml [hunk 1]
```diff
@@ -107,6 +107,10 @@ spec:
           ports:
             - name: {{ .Values.api.service.internalPortName }}
               containerPort: {{ .Values.api.service.internalPort }}
+            {{- if .Values.api.http.services.core.http.enabled }}
+            - name: {{ .Values.api.name }}-hook
```
Comment (brasseld) [inline apim/3.x/templates/api/api-deployment.yaml:111]: Not sure what's the meaning of `-hook` here. Shouldn't it be something like `-tech-api` ?

**Review by brasseld — CHANGES_REQUESTED** [review]
See comments

File: apim/3.x/templates/api/api-deployment.yaml (hunk 1 above)
Comment (gaetanmaisse) [reply apim/3.x/templates/api/api-deployment.yaml:111]: To be consistent with https://github.com/gravitee-io/helm-charts/blob/70d07a94244a55a79879cba445100b610cf977e4/apim/3.x/templates/gateway/gateway-deployment.yaml#L128

Comment (brasseld) [reply apim/3.x/templates/api/api-deployment.yaml:111]: Ok, I think that doesn't make sense. This one has been added in the past to the gateway for an other purpose (kubernetes hook), and I think it's been done wrongly at that time.

**Comment by brasseld** [comment]
LGTM, did you try to deploy it?

**Pull request merged**